Mathlib/Tactic.lean

```diff
@@ -7,6 +7,7 @@ import Mathlib.Tactic.Backtracking
 import Mathlib.Tactic.Basic
 import Mathlib.Tactic.ByContra
 import Mathlib.Tactic.Cache
+import Mathlib.Tactic.CancelDenoms
```

Comment:
Is this PR somehow have a wrong configuration / base branch, that this line exists?

Reply:
Yes, I couldn't tell why this line was not picked up by my merge with master, and I had to add it back by hand to make the linter happy. I will merge and see if bors complains.